wbooks/src/App/screens/LoginPage/components/LoginForm/layout.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { reduxForm, Field } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import { required, email, passMinLength, passMaxLength } from '../../../../utils/formValidator';
+
+const renderField = ({ input, label, type, meta: { touched, error, warning } }) => (
```

Comment:
Right now you are not using this anywhere else, but it's a good idea to extract this and the textarea one to a component.